tikv-configuration-file.md

```diff
@@ -1543,3 +1543,35 @@ For pessimistic transaction usage, refer to [TiDB Pessimistic Transaction Mode](
 + Enables the in-memory pessimistic lock feature. With this feature enabled, pessimistic transactions try to store their locks in memory, instead of writing the locks to disk or replicating the locks to other replicas. This improves the performance of pessimistic transactions. However, there is a still low probability that the pessimistic lock gets lost and causes the pessimistic transaction commits to fail.
 + Default value: `true`
 + Note that `in-memory` takes effect only when the value of `pipelined` is `true`.
+
+## quota
+
+Configuration items related to Quota Limiter.
+
+Suppose that your machine deployed TiKV has limited resources, for example, 4v CPU, 16 G memory. In this case, if TiKV foreground processes requests in too large volumes, the foreground occupies the CPU resources for processing background requests, which causes performance degradation. To avoid this issue, you can use the quota-related configuration items to limit the CPU resources used by the foreground. When requests trigger the Quota Limiter feature, the requests are forced to wait for a while to free up CPU resources. The exact waiting time depends on the volume of the requests, and the maximum waiting time is no longer than the value of [`max-delay-duration`](#max-delay-durationnew-in-v600).
```

Comment:
```suggestion
Suppose that your machine deployed TiKV has limited resources, for example, 4v CPU, 16 G memory. In this case, if the amount of requests processed by TiKV in the foreground is too large, the CPU resources for background processing are occupied by the foreground, causing TiKV performance to degrade. To avoid this issue, you can use the quota-related configuration items to limit the CPU resources used by the foreground. When requests trigger the Quota Limiter feature, the requests are forced to wait for a while to free up CPU resources. The exact waiting time depends on the volume of the requests, and the maximum waiting time is no longer than the value of [`max-delay-duration`](#max-delay-durationnew-in-v600).
```